storage/migrations/01_runtimes.up.sql

```diff
@@ -86,19 +86,19 @@ CREATE TABLE chain.runtime_transactions
   oasis_encrypted_result_data BYTEA,
 
   -- Error information.
-  success       BOOLEAN,  -- NULL means success is unknown (can happen in confidential runtimes)
+  success       BOOLEAN,  -- NULL means success is unknown (can happen in confidential runtimes, or for 'consensusaccounts' transactions which whose action is known only in the next round)
```

Comment:
Not convinced if it's better to re-use the existing `success` field, or introduce a new additional `status` field. With the new field, we would be able to differentiate between "transaction was successful, but the event in the next round failed", but I don't think we need that. One can get that info by looking at events.

Unless there's good reasoning against it, I would just reuse the `status` field.

Reply:
Agreed 👍